assets/css/admin.scss

```diff
@@ -1129,3 +1129,26 @@ h2 .nav-tab-active, nav .nav-tab-active {
 .jm-logo__wrapper:hover .jm-logo__letters {
 	transform: rotateY(90deg);
 }
+
+/**
+ * Adjust upsells styles
+ */
+.wpjm-addon-upsell__badge {
+	display: inline-block;
+	vertical-align: middle;
+	box-sizing: border-box;
+	margin: 1px 0 -1px 2px;
+	padding: 0 3px; /* Reduced padding */
+	min-width: 15px; /* Slightly smaller */
+	height: 15px; /* Slightly smaller */
+	border-radius: 5px; /* Less rounded */
+	background-color: #6C737C; /* More neutral color */
+	color: #fff;
+	font-size: 10px; /* Smaller font size */
+	line-height: 1.6;
+	text-align: center;
+	z-index: 26;
+	/* Move upsell to the right */
+	position: absolute;
+	right: 10px;
```

Comment:
A few changes for perfecting alignment, plus smaller font size seems to work better:

```css
right: 10px;
top: 0;
bottom: 0;
margin: auto 0;
display: flex;
align-items: center;
  font-size: 8px;
```

Reply:
I added these suggestions in ee40e8b8fae2f02699a74ed876a2fa6010e1156f, but I must say that at least in 1080p (which is the resolution that I use), the font-size looks VERY small:

![Upsells based on Peter's suggestions](https://github.com/Automattic/WP-Job-Manager/assets/529864/5584bc9f-6629-4a9d-a5b0-1ec07a05f0b0)

I think that with 10px it looks better (while also applying all your other suggestions):

![Upsells with 10px font size](https://github.com/Automattic/WP-Job-Manager/assets/529864/15034692-58cf-4b82-8a61-5429532d4a8d)

WDYT?